### DISCUSSION TITLE
Request for spanish translation README

### DISCUSSION THREAD

**Discussion by moradebianchetti81** [fastify/fastify#5456] [General]
hi, i noticed that the project readme file is currently only in english. I wanted to ask if there is a spanish version of the file.
If this is not in progress, I would like to provide this documentation.
could anyone give me an idea if a version of this is being developed or if translation contributions are accepted?

**Comment by Uzlopak** [comment]
What is the goal of having the readme in spanish, if we can not ensure that we keep it up to date, as not every contributor can write spanish. Also spanish has more writing styles... Should we use argentinian spanish? gallego spanish? ecuadorian spanish? philipinian spanish?

**Comment by moradebianchetti81** [comment]
the objective of having the readme in other languages ​​is to make the project more accessible and inclusive for contributors, in this case spanish-speaking. While maintaining multiple language versions can pose challenges, providing translations can benefit a diverse user base. Besides, the readme is not a resource that changes that frequently.
You can adopt neutral spanish, in short I think the objective would be to provide clearer and more understandable documentation for users

**Reply by Uzlopak** [reply]
Please repeat this comment in https://github.com/fastify/website/issues/219#issuecomment-2104302785

**Comment by Uzlopak** [comment]
We should bundle the discussion in 

https://github.com/fastify/website/issues/219#issuecomment-2104302785
